doc/source/timeseries.rst

```diff
@@ -2129,11 +2129,13 @@ These can easily be converted to a ``PeriodIndex``:
 Time Zone Handling
 ------------------
```

Comment:
I would split timezones off into a separate section entirely (but can do that in a separate PR)

Reply:
Agreed. I think there's enough content in `timeseries.rst` to split into `Timestamp` (and timezones), `Timedelta`, `Period` and `Offset`. Will work on those in separate PRs